oadr/oadr/manager/VENManagerConfig.h

```diff
@@ -30,6 +30,15 @@ struct ttls
 	ttls();
 };
 
+struct tsignature
+{
+	std::string certPath;
+	std::string signingKeyPath;
+	std::string caBundlePath;
+
+	tsignature();
```

Comment:
Nitpick: Even if it is possible, I don't like (It is 100% personal preference, feel free to ignore) constructors/methods on structs.  If the project uses the same pattern, it is fine.

Reply:
My intention here was to keep consistency with the rest of the code in this file.

And yes, in C++ structs and classes are very similar, there are minor differences, like members of classes are all private by default, while in structs they are are public by default.